TestHandlerImplementsHTTPCloseNotify is flaky

It occasionally fails, see the log:

```
=== RUN   TestHandlerImplementsHTTPCloseNotify
--- FAIL: TestHandlerImplementsHTTPCloseNotify (1.03s)
    server_test.go:554: HTTP2Log got
        	"ended\nended\nended\n"
        want
        	"ended\nended\nended\nended\n"
```

--- a/plugin/ochttp/server_test.go
+++ b/plugin/ochttp/server_test.go
@@ -543,7 +543,7 @@ func TestHandlerImplementsHTTPCloseNotify(t *testing.T) {
 	}
 
 	// Wait for a couple of milliseconds for the GoAway frames to be properly propagated
-	<-time.After(150 * time.Millisecond)
+	<-time.After(200 * time.Millisecond)
 
 	wantHTTP1Log := strings.Repeat("ended\n", len(transports))
 	wantHTTP2Log := strings.Repeat("ended\n", len(transports))

